ft-sdk/src/auth/mod.rs

```diff
@@ -122,19 +144,57 @@ pub fn ud(
 
     Ok(Some(ft_sys::UserData {
         id,
-        identity: identity.expect("user fetched from session cookie must have identity"),
+        identity: identity.ok_or_else(|| UserDataError::IdentityNotFound)?,
         name: data.name.unwrap_or_default(),
         email,
         verified_email: !data.verified_emails.is_empty(),
     }))
 }
 
+/// Check if debug user data is available, return it if found.
+fn get_debug_ud() -> Option<ft_sys::UserData> {
+    match ft_sys::env::var("DEBUG_LOGGED_IN".to_string()) {
+        Some(debug_logged_in) => {
+            let mut debug_logged_in = debug_logged_in.splitn(4, ' ');
+            Some(ft_sys::UserData {
+                id: debug_logged_in.next().unwrap().parse().unwrap(),
+                identity: debug_logged_in.next().unwrap_or_default().to_string(),
+                name: debug_logged_in
+                    .next()
+                    .map(|v| v.to_string())
+                    .unwrap_or_default(),
+                email: debug_logged_in
+                    .next()
+                    .map(|v| v.to_string())
+                    .unwrap_or_default(),
+                verified_email: true,
+            })
+        }
+        None => None,
+    }
+}
+
+// This is hack to keep mobile number as email.
+pub fn mobile_to_email(mobile_number: &str) -> String {
+    format!("{mobile_number}@mobile.fifthtry.com")
+}
```

Comment:
Nopes, this is not final enough for us to make it part of ft-sdk, currently this is a hack for one site. Remove it from here.

Reply:
Done